jetpack.php

```diff
@@ -139,6 +139,19 @@ function jetpack_admin_missing_autoloader() { ?>
 	<?php
 }
 
+/**
```

Comment:
Thanks for that. As the file is getting bigger, it's nice to get some documentation in there.

Reply:
Indeed, thanks! Better documentation at this point could be very useful to folks who dig into this code for the first time.